code/mission/import/xwingmissionparse.cpp

```diff
@@ -109,6 +109,9 @@ int xwi_determine_arrival_cue(const XWingMission *xwim, const XWMFlightGroup *fg
 	}
 	else
 		return Locked_sexp_true;
+	
+	if (fg->arrivalEvent == XWMArrivalEvent::ae_mission_start)
+		return Locked_sexp_true;
```

Comment:
It's not necessary to add this code here because it was already included in the other PR.  Since the other PR was merged, this code now exists in master.  Once you merge your branch and update master, you'll see it too.